app/views/posts/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_for :post, url: user_posts_path(@user) do |f| %>
+  Post Title: <%= f.text_field :title %><br />
+  Post Details: <%= f.text_area :content %><br />
+  <%= f.submit %>
+<% end %>
```

Comment:
When adding a post title the title should reflect on the page. Currently, the app adds automated titles (Post#1, Post#2, Post#3, etc.) no matter what text I enter on the title input field when creating a post.

- Kindly make sure that the views display the correct title that is created by the user.